projects/packages/tracking/src/class-tracking.php

```diff
@@ -87,6 +87,19 @@ public function ajax_tracks() {
 		wp_send_json_success();
 	}
 
+	/**
+	 * Register script necessary for tracking.
+	 */
+	public function register_tracks_scripts() {
```

Comment:
Lets update this to a static function. There is no reason to need to instantiate a Tracking class.